docker/develop/Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM ubuntu:16.04
 # number of concurrent threads during build
 # usage: docker build --build-arg PARALLELISM=8 -t name/name .
 ARG PARALLELISM=1
+ARG CMAKE_BUILD_TYPE=Release
```

Comment:
This docker image is used for building and packaging of release builds. Current cmake is written in such a way, that dependencies first are searched in OS. When iroha is built inside this image, then all dependencies can be found and these dependencies are used for packaging (they are packaged inside deb/rpm/tgz), thus making *potential* benchmarks irrelevant.